zerver/webhooks/freshping/view.py

```diff
@@ -16,9 +16,10 @@
 Error code: {http_status_code}.
 """.strip()
 FRESHPING_MESSAGE_TEMPLATE_UP = "{request_url} is back up and no longer unreachable."
+ALL_EVENT_TYPES = ["Reporting Error", "Available"]
```

Comment:
We probably want to convert these to something that's more of an identifier, if upstream doesn't provide one.